### PR TITLE
Update Project section

### PR DESCRIPTION
**Pull request by mishmosh**
This should create an MVP Project section:

- Removed irrelevant sections in config.js
- Updated links and text

### CODE REVIEW

File: docs/project/README.md [hunk 1]
```diff
@@ -1,45 +1,31 @@
...
-## Implementation status
+The latest timing information is available in the [Filecoin Mainnet Roadmap](https://app.instagantt.com/shared/s/1152992274307505/latest).
 
-Want to know the current state of your favorite IPFS feature? See the [current implementation status](https://github.com/ipfs/ipfs/blob/master/IMPLEMENTATION_STATUS.md) for go-ipfs and js-ipfs, listed by commands and endpoints.
+## Filecoin specification
 
-## IPFS specifications
-
-[Technical specifications](https://github.com/ipfs/specs) for the IPFS protocol and its associated subsystems.
+[Technical specification](https://github.com/filecoin-project/specs) for the Filecoin protocol and its associated subsystems.
```
Comment (terichadbourne) [inline docs/project/README.md]:
```suggestion
View the [technical specification](https://github.com/filecoin-project/specs) for the Filecoin protocol and its associated subsystems.
```

Just turning this into a sentence to match the other sections.

File: docs/project/README.md [hunk 2]
```diff
@@ -1,45 +1,31 @@
...
 
 ## Research
 
-Learn more about the exploratory research work and prototyping being done for inclusion in IPFS by exploring our [research repo on GitHub](https://github.com/ipfs/research).
-
-## IPFS team org planning
-
-See how IPFS team planning, management and coordination happens, and learn how to take part in the weekly open call, by visiting the [IPFS team management GitHub repo](https://github.com/ipfs/team-mgmt).
+Learn about the ongoing work to design the predicates that enable Filecoin by exploring our [Filecoin Research website](https://research.filecoin.io/).
```
Comment (terichadbourne) [inline docs/project/README.md]: Is there a word other than `predicates` that would work here? I suspect many folks  only know one meaning of this word:
🎶 _Mr. Morton is the subject of the sentence, and what the predicate says, he does._ 🎶

File: docs/project/README.md [hunk 3]
```diff
@@ -1,45 +1,31 @@
...
 
 ## Related projects
 
-IPFS is a highly modular project that is itself made out of many different protocols and tools. Learn more about the [IPFS-related projects](/project/related-projects/) under the overall support of Protocol Labs.
-
-## Contribute to IPFS
-
-Thousands of people contribute to IPFS from all over the world — and that can include you! No matter your areas of interest or expertise, there are a number of ways that you can make an impact on the future of the Internet by [contributing to IPFS](/project/contribute/)
+Filecoin is a highly modular project that is itself made out of many different protocols and tools. Learn more about the [Filecoin-related projects](/project/related-projects/) under the overall support of Protocol Labs.
```
Comment (terichadbourne) [inline docs/project/README.md]:
```suggestion
Filecoin is a highly modular project that is itself made out of many different protocols and tools. Learn more about the [Filecoin-related projects](/project/related-projects/) supported by Protocol Labs.
```

File: docs/project/related-projects.md [hunk 4]
```diff
@@ -2,27 +2,21 @@
...
-# TODO: REPLACE/REUSE CONTENT
-
 # Related projects
 
-IPFS is a highly modular project that is itself made out of many different protocols and tools. Many of these exist as their own projects under the overall support of [Protocol Labs](https://protocol.ai). Learn more about them below.
-
-## IPLD
-
-IPLD is the data model of the content-addressable web. It allows us to treat all hash-linked data structures as subsets of a unified information space, unifying all data models that link data with hashes as instances of IPLD. Learn more at [https://ipld.io/](https://ipld.io/).
+Filecoin is a highly modular project that is itself made out of many different protocols and tools. Many of these exist as their own projects under the overall support of [Protocol Labs](https://protocol.ai). Learn more about them below.
```
Comment (terichadbourne) [inline docs/project/related-projects.md]:
```suggestion
Filecoin is a highly modular project that is itself made out of many different protocols and tools. Many of these exist as their own projects, supported by [Protocol Labs](https://protocol.ai). Learn more about them below.
```

File: docs/project/README.md (hunk 3 above)
Comment (mishmosh) [reply docs/project/README.md]: Nice, that phrase was feeling a bit too broad.

File: docs/project/README.md (hunk 2 above)
Comment (mishmosh) [reply docs/project/README.md]: Rewrote the sentence entirely :)

**Comment by mishmosh** [comment]
> And just confirming you've intentionally skipped adding content to the history page for now?

Yes, there's no easily existing content, so leaving that for a separate PR.

**Pull request merged**